src/app/login-basic/authentication-basic.service.ts

```diff
@@ -1,36 +1,14 @@
 import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { tap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
 import { User } from './user';
-import { environment } from '../../environments/environment';
-import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { map } from 'rxjs/operators';
-import { Observable } from 'rxjs/internal/Observable';
 
-@Injectable()
+@Injectable({ providedIn: 'root' })
 export class AuthenticationBasicService {
+  private authUrl = '/api/auth';
```

Comment:
You cannot use a relative URL for the API, it should be absolute as it is in a different server and should be configurable depending on where it is deployed. Use "${environment.API}/identity" instead